### ISSUE TITLE
Deno LSP running via VSCode extension crashed due to unwrap() on Err

### ISSUE THREAD

**Issue by alexandros-megas** [denoland/deno#15680]
Working on a relatively small TS web app project (roughly 10 source files), and got the following error after creating a file and typing some code:

```
Unable to send result to client.
Unable to get completion info from TypeScript: Error: Debug Failure. False expression.
    at getCompletionEntryCodeActionsAndSourceDisplay (deno:cli/tsc/00_typescript.js:133230:22)
    at Object.getCompletionEntryDetails (deno:cli/tsc/00_typescript.js:133168:30)
    at Object.getCompletionEntryDetails (deno:cli/tsc/00_typescript.js:165282:35)
    at serverRequest (deno:cli/tsc/99_main_compiler.js:717:27)
    at [deno:cli\lsp\tsc.rs:3300:27]:1:12
[Error - 10:53:17 PM] Request completionItem/resolve failed.
  Message: Internal error
  Code: -32603 

============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: windows x86_64
Version: 1.23.4
Args: ["C:\\ProgramData\\chocolatey\\lib\\deno\\deno.exe", "lsp"]

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("invalid value: 3, expected one of: 0, 1, 2, 4, 8, 16, 32", line: 0, column: 0)', cli\lsp\tsc.rs:153:54
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[Error - 10:53:27 PM] The Deno Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
```

My .vscode/settings.json file just has the bare "deno.enable" set to `true`.
My Deno extension version is v3.13.1

Let me know if there's any more info you'd like me to provide.

**Comment by 0f-0b** [comment]
Seems like #15287 which was fixed in 1.24.1.

**Comment by kitsonk** [comment]
Duplicate of #15287
